app/models/manageiq/providers/ibm_cloud/vpc/manager_mixin.rb

```diff
@@ -11,7 +11,7 @@ def supported_auth_attributes
 
   # Return a cloudtools vpc object.
   # @param options [Hash] Hash of options. Default to an empty Hash.
-  # @return [ManageIQ::Providers::IbmCloud::CloudSDK::Vpc]
+  # @return [ManageIQ::Providers::IbmCloud::CloudTools::Vpc]
```

Comment:
singular or plural?

Reply:
Plural for the module. Singular for the class. Normally I would use all uppercase for the organizing module but Zeitwerk doesn't support that.

Reply:
classes are modules in Ruby, so if the only purpose if for subclasses, you can also just nest them under the parent class.  I don't really understand the need for a distinction between the module and the class.

Reply:
It's not for OOP subclassing. It is for organizing. In Vpc the method instances returns VpcSdk::Instances.new(:parent => self).

When you say nest do you mean define a concrete class within the Vpc class?. I don't want to do that. I want a separate file for each all grouped under the same organizing namespace.